wordpress-mobile/WordPressKit-iOS
# Merge feature branch for Site Design Screen Improvements project

### Description

Included PRs:
- https://github.com/wordpress-mobile/WordPressKit-iOS/pull/507

### Testing Details

The [WPiOS feature branch](https://github.com/wordpress-mobile/WordPress-iOS/blob/30d853aab5d2731a1b137fb1ccec9388d641d1e2/Podfile#L53) is pointed to 88d45f4712738226b6aedbff24064416ac35b4a4.

1. Compile the [WPiOS feature branch](https://github.com/wordpress-mobile/WordPress-iOS/pull/18740)
2. Navigate to the Site Design screen
3. Confirm that site designs are loaded 

- [x] I have considered updating the `version` in the `.podspec` file.
